processing/src/test/java/org/apache/druid/query/groupby/epinephelinae/BufferHashGrouperTest.java

```diff
@@ -140,8 +140,8 @@ public void testGrowing()
   public void testGrowing2()
   {
     final TestColumnSelectorFactory columnSelectorFactory = GrouperTestUtil.newColumnSelectorFactory();
-    final Grouper<Integer> grouper = makeGrouper(columnSelectorFactory, 2_000_000_000, 2);
-    final int expectedMaxSize = NullHandling.replaceWithDefault() ? 40988516 : 39141224;
+    final Grouper<Integer> grouper = makeGrouper(columnSelectorFactory, 250_000_000, 2);
```

Comment:
Should you add a comment to explain why this value is significant? Also if you're feeling ambitious, maybe would be nice to rename tests to something like `testGrowingOverflow` since that is what I think this is testing, and maybe `testGrowingMax` for `testGrowing3`?